src/dodal/parameters/experiment_parameter_base.py

```diff
@@ -1,7 +1,15 @@
 from abc import ABC, abstractmethod
 
+from pydantic import BaseModel
+
+
+class AbstractExperimentParameterBase(BaseModel, ABC):
+    pass
+
+
+class AbstractExperimentWithBeamParams(AbstractExperimentParameterBase):
+    transmission_fraction: float
```

Comment:
Wouldn't we always need a transmission fraction? Could this just be in the original `AbstractExperimentParameterBase`?

Reply:
we might want to expose endpoints which only do things without beam, e.g. optical centring, or load a sample only, and those don't need transmission